packages/packagers/js/src/DevPackager.js

```diff
@@ -116,6 +121,20 @@ export class DevPackager {
         wrapped += JSON.stringify(deps);
         wrapped += ']';
 
+        if (
+          this.bundle.env.isNode() &&
+          asset.meta.has_node_replacements === true
+        ) {
+          const relPath = normalizeSeparators(
+            path.relative(
+              this.bundle.target.distDir,
+              path.dirname(asset.filePath),
+            ),
+          );
+          wrapped = wrapped.replace('$parcel$dirnameReplace', relPath);
+          wrapped = wrapped.replace('$parcel$filenameReplace', relPath);
```

Comment:
Is there a reason we have two different variables that get replaced with the same path? Were these meant to be different, or do we only need one of them?